zhaquirks/gledopto/glsd001.py

```diff
@@ -31,10 +31,10 @@ class LevelControlNoReply(NoReplyMixin, CustomCluster, LevelControl):
 
 
 class GledoptoGlSd001(CustomDevice):
```

Comment:
Maybe also rename the class name to something like `GledoptoDimmer`, as well as renaming the file name(?)

Reply:
Renamed in file in [657812e](https://github.com/zigpy/zha-device-handlers/pull/2632/commits/657812e5dd37d29c46c1f4d2cb76428baded8cae)
Renamed class in [e6f21c2](https://github.com/zigpy/zha-device-handlers/pull/2632/commits/e6f21c209ba049197da20c71b313eb95eabb183e)